src/current/releases/unsupported-versions.md

```diff
@@ -15,7 +15,7 @@ docs_area: releases
 {% assign released_versions = site.data.releases | map: "major_version" | uniq | reverse %}
```

Comment:
I think this page needs a callout on the top to clarify that the versions aren't supported. Try this:

```
{{site.data.alerts.callout_danger}}
The CockroachDB versions on this page are no longer supported. For more information, refer to [Release Support Policy]({% link releases/release-support-policy.md %}#unsupported-versions). To download and learn about currently supported CockroachDB versions, refer to [CockroachDB Releases]({% link releases/index.md %}).
{{site.data.alerts.end}}
```